src/test/java/org/minions/devfund/angela/SquarelotronTest.java

```diff
@@ -0,0 +1,120 @@
+package org.minions.devfund.angela;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test for {@link Squarelotron} .
+ */
+public class SquarelotronTest {
+
+    private Squarelotron squarelotron;
+
+    /**
+     * Preconditions.
+     */
+    @Before
+    public void setup() {
+        final int size = 4;
+        squarelotron = new Squarelotron(size);
+    }
+
+    /**
+     * Verifies if certain position is part of the ring.
+     */
+    @Test
+    public void testIsPositionPartOfTheRing() {
+        assertTrue("The position isn't part of the ring", squarelotron.isPartOfTheRing(1, 0, 0));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "The position isn't part of the ring" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)